tests/cephfs/cephfs_case_sensitivity/case_sensitivity_disruptive.py

```diff
@@ -0,0 +1,138 @@
+        if not attr_util.validate_alternate_name(alter_dict, rel_child_dir):
+            log.error("Validation failed for alternate name")
+
+        log.info(
+            "\n"
+            "\n---------------***************-----------------------------------"
+            "\n    Usecase 1: Persistence after MDS reboot                      "
+            "\n---------------***************-----------------------------------"
+        )
+
+        num_of_osds = config.get("num_of_osds")
+        fs_util.runio_reboot_active_mds_nodes(
+            fs_util,
+            ceph_cluster,
+            fs_name,
+            client1,
+            num_of_osds,
+            build,
+            child_dir,
+        )
+
+        assert attr_util.get_charmap(client1, child_dir).get("casesensitive") is False
+        assert attr_util.get_charmap(client1, child_dir).get("normalization") == "nfkd"
+        assert attr_util.get_charmap(client1, child_dir).get("encoding") == "utf8"
+
+        log.info("Capture the mds states after rebooting")
+        mds_info_after_for_fs1 = fs_util.get_mds_states_active_standby_replay(
+            fs_name, client1
+        )
+        log.info(f"MDS Info for {fs_name} After Reboot: {mds_info_after_for_fs1}")
+
+        log.info("Validating alternate name for %s", rel_child_dir)
+        alter_dict_1 = attr_util.fetch_alternate_name(client1, fs_name, "/")
+        if not attr_util.validate_alternate_name(alter_dict_1, rel_child_dir):
+            log.error("Validation failed for alternate name")
+
+        log.info("Passed: Attribute persisted after MDS reboot")
+
+        log.info(
+            "\n"
+            "\n---------------***************-----------------------------------"
+            "\n    Usecase 2: Persistence after remount                         "
+            "\n---------------***************-----------------------------------"
+        )
+
+        log.info("Unmounting fuse client:")
+        cmd = "fusermount -u {} -z".format(fuse_mounting_dir)
+        client1.exec_command(sudo=True, cmd=cmd)
+
+        fs_util.fuse_mount(
+            [client1], fuse_mounting_dir, extra_params=" --client_fs {}".format(fs_name)
+        )
+
+        assert attr_util.get_charmap(client1, child_dir).get("casesensitive") is False
+        assert attr_util.get_charmap(client1, child_dir).get("normalization") == "nfkd"
+        assert attr_util.get_charmap(client1, child_dir).get("encoding") == "utf8"
+
+        log.info("Passed: Attribute persisted after remount")
+
+    except Exception as e:
+        log.error("Test execution failed: {}".format(str(e)))
+        log.error(traceback.format_exc())
+        return 1
+
+    finally:
+        fs_util.client_clean_up(
+            "umount", fuse_clients=[client1], mounting_dir=fuse_mounting_dir
+        )
+        fs_util.remove_fs(client1, fs_name)
+
+    log.info("Disruptive Case Sensitive use cases completed")
+    return 0
```

Comment:
can you move this return statement to line no : 125 
After this log.info("Passed: Attribute persisted after remount") 

Same for all the other cases